spec/features/admin/questionnaire_spec.rb

```diff
@@ -15,66 +15,38 @@
     it "works" do
       # create
       visit admin_loan_tab_path(loan, 'questions')
-      fill_in('response_set[summary[text]]', with: 'kittens jumping on rainbows')
-      click_button 'Save Responses'
-      expect(page).to have_content 'successfully created'
-      expect(page).to have_content 'kittens jumping on rainbows'
-      expect(criteria.summary.text).to eq 'kittens jumping on rainbows'
 
-      # edit
-      click_link('Edit Responses')
-      fill_in('response_set[summary[text]]', with: 'sexy unicorns')
-      click_button 'Save Responses'
-      expect(page).to have_content 'successfully updated'
-      expect(page).to have_content 'sexy unicorns'
-      expect(page).not_to have_content 'Warning'
-      expect(criteria.summary.text).to eq 'sexy unicorns'
-
-      # delete
-      accept_confirm { click_link('Delete All Responses') }
-      expect(page).to have_content 'successfully deleted'
-      expect(page).not_to have_content 'sexy unicorns'
-      # After deletion, should be in edit mode
-      expect(page).not_to have_content 'Edit Responses'
-      expect(page).to have_selector 'input[value="Save Responses"]'
+      # check that edit-all button turns on edit mode
+      expect(page).not_to have_content("Now editing")
+      page.find(".edit-all", match: :first).click
+      expect(page).to have_content("Now editing")
+
+      # cancel button is visible in edit mode
+      edit_bar = page.find("#editBar")
+      edit_bar.find(".cancel-edit").click()
+      expect(page).not_to have_content("Now editing")
+
+      # save changes button is visible in edit mode
+      page.find(".edit-all", match: :first).click
+      expect(page).to have_content("Now editing")
+      click_button 'Save Changes'
```

Comment:
I'd like us to be consistent about parenthesis usage please. Would you mind adding some here. Later on I'll likely propose we add some rubocop rules for this so it can be caught while we are all coding and not have to take up time in PRs.